src/views/edit/edit-provider-view.vue

```diff
@@ -451,6 +453,7 @@
         if (this.isEdit) {
           document.title = this.$i18n.tc('component.provider.label') + ' – ' + this.allNames.name
         }
+        languageServices.fetchLanguagesList(createCancelToken().token)
```

Comment:
Yes, there is some redundancy here. For now, it would be sufficient to load languages in the watcher of `$i18n.locale`. ...which I would prefer, since a change of the UI language would not be reflected in the languages list otherwise.
Anyhow, I felt, it would be good to have the languages app-wide as well.
Since programming is not about feelings: should I better remove the statement from `app.vue`?